test/units/modules/remote_management/oneview/hpe_test_utils.py

```diff
@@ -15,12 +15,31 @@
 # You should have received a copy of the GNU General Public License
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
 
+import pytest
 import yaml
+
 from mock import Mock, patch
 from oneview_module_loader import ONEVIEW_MODULE_UTILS_PATH
 from hpOneView.oneview_client import OneViewClient
 
 
+@pytest.fixture
```

Comment:
These fixtures should likely be inside `conftest.py`

Reply:
Good!